evalml/tests/component_tests/test_time_series_regularizer.py

```diff
@@ -23,13 +23,29 @@ def get_df(dates):
     return reg_X, reg_y
 
 
+def get_unstacked_df(dates):
+    reg_X = pd.DataFrame()
+
+    reg_X["dates"] = dates
+    reg_X["feature_a_0"] = [int(i) for i in range(len(dates))]
+    reg_X["feature_a_1"] = [int(i) for i in range(len(dates), 0, -1)]
+    reg_X["feature_b_0"] = [int(i) for i in range(len(dates) * 2, 0, -2)]
+    reg_X["feature_b_1"] = [int(i) for i in range(0, len(dates) * 2, 2)]
+
+    reg_y = pd.DataFrame()
+    reg_y["target_0"] = [int(i) for i in range(len(dates))]
+    reg_y["target_1"] = [int(i) for i in range(len(dates), 0, -1)]
```

Comment:
You can simplify all the `[int(i) for i in range(<params>)]` to be just `list(range(<params>))`!